src/main/bin/resolveOptions.ts

```diff
@@ -89,6 +89,11 @@ export function resolveOptions(args: Array<string>): IMakeOptions {
                 index += 2
                 break
 
+            case '--file-per-type':
+                options.filePerType = true
+                index += 1
```

Comment:
is this the right way to handle a boolean flag?

Reply:
Yeah, I think that's good. But if we go this way I'm not even sure we need the flag. This can just become the default behavior. It really shouldn't be breaking change for consumers.